src/constants.h

```diff
@@ -119,6 +119,15 @@ PRODBETAEXPR(
     "97e8ba9cf12fb3de53cc42a4e6577ed64df493c247b414fea036818d3823560e",
     "3c01446abe9036cea9a09acaa3a520ac628f20a7ae32ce861cb2efb70fa0c745");
 
+PRODBETAEXPR(const char*, relayUrl, "https://relay.firefox.com",
+             "https://stage.fxprivaterelay.nonprod.cloudops.mozgcp.net");
+
+PRODBETAEXPR(const char*, bundleUpgradeUrl,
```

Comment:
This makes me nervous. Would it be better to bounce folks through Guardian and let Guardian redirect?